app/poros/week.rb

```diff
@@ -0,0 +1,26 @@
+class Week
+  def self.determine(event_data)
+    found_event = get_valid_week(split_events(event_data))
+    get_week_number(found_event)
+  end
+
+  private
+
+    def self.split_events(event_data)
+      event_data.map{ |event| event[:summary].split }
```

Comment:
events are split so the number of words can be counted
ie: "Week 4 Final Assessment".split = ["Week", "4", "Final", "Assessment"]